templates/EbookPlaceholderForm.php

```diff
@@ -60,64 +97,133 @@
+			name="collection-name-1"
+			list="collection-names"
+			value="<? if(isset($ebook->CollectionMemberships)){ ?><?= Formatter::EscapeHtml($ebook->CollectionMemberships[0]->Collection->Name) ?><? } ?>"
+		/>
 	</label>
 	<fieldset>
 		<label>
 			<span>Number in collection</span>
 			<input
 				type="text"
-				name="collection-membership-sequence-number"
+				name="sequence-number-collection-name-1"
 				inputmode="numeric"
-				pattern="[0-9]{1,3}"/>
+				pattern="[0-9]{1,3}"
+				value="<? if(isset($ebook->CollectionMemberships)){ ?><?= Formatter::EscapeHtml((string)$ebook->CollectionMemberships[0]->SequenceNumber) ?><? } ?>"
+			/>
 		</label>
 	</fieldset>
 </fieldset>
+<details>
+	<summary>Additional collections</summary>
+	<fieldset>
+		<label>
+			<span>Second Collection</span>
+			<datalist id="collection-names">
+				<? foreach(Collection::GetAll() as $collection){ ?>
+					<option value="<?= Formatter::EscapeHtml($collection->Name) ?>"><?= Formatter::EscapeHtml($collection->Name) ?></option>
+				<? } ?>
+			</datalist>
+			<input
+				type="text"
+				name="collection-name-2"
+				list="collection-names"
+				value="<? if(isset($ebook->CollectionMemberships)){ ?><?= Formatter::EscapeHtml($ebook->CollectionMemberships[0]->Collection->Name) ?><? } ?>"
+			/>
+		</label>
+		<fieldset>
+			<label>
+				<span>Number in collection</span>
+				<input
+					type="text"
+					name="sequence-number-collection-name-2"
+					inputmode="numeric"
+					pattern="[0-9]{1,3}"
+					value="<? if(isset($ebook->CollectionMemberships)){ ?><?= Formatter::EscapeHtml((string)$ebook->CollectionMemberships[0]->SequenceNumber) ?><? } ?>"
+				/>
+			</label>
+		</fieldset>
+	</fieldset>
+	<fieldset>
+		<label>
+			<span>Third Collection</span>
+			<datalist id="collection-names">
+				<? foreach(Collection::GetAll() as $collection){ ?>
+					<option value="<?= Formatter::EscapeHtml($collection->Name) ?>"><?= Formatter::EscapeHtml($collection->Name) ?></option>
+				<? } ?>
+			</datalist>
+			<input
+				type="text"
+				name="collection-name-3"
+				list="collection-names"
+				value="<? if(isset($ebook->CollectionMemberships)){ ?><?= Formatter::EscapeHtml($ebook->CollectionMemberships[0]->Collection->Name) ?><? } ?>"
+			/>
+		</label>
+		<fieldset>
+			<label>
+				<span>Number in collection</span>
+				<input
+					type="text"
+					name="sequence-number-collection-name-3"
+					inputmode="numeric"
+					pattern="[0-9]{1,3}"
+					value="<? if(isset($ebook->CollectionMemberships)){ ?><?= Formatter::EscapeHtml((string)$ebook->CollectionMemberships[0]->SequenceNumber) ?><? } ?>"
+				/>
+			</label>
+		</fieldset>
+	</fieldset>
+</details>
 <fieldset>
 	<legend>Corpus details</legend>
 	<label>
 		<span>On the SE wanted list?</span>
 		<input
 			type="checkbox"
-			name="ebook-placeholder-is-wanted"/>
+			name="ebook-placeholder-is-wanted"
+			<? if(isset($ebook->EbookPlaceholder) && $ebook->EbookPlaceholder->IsWanted){ ?>checked="checked"<? } ?>
+		/>
```

Comment:
You can make the following fields hidden if it is not on the wanted list, that will make the form a lot cleaner. Put them in a `<fieldset>` and use this CSS:

```css
input[name="ebook-placeholder-is-wanted"] + fieldset{
    display: none;
}
input[name="ebook-placeholder-is-wanted"]:checked + fieldset{
    display: block;
}

Reply:
You're a wizard, thanks! I used slightly more-specific CSS and a different structure because I was getting a lot of conflicts with some rules in `core.css`, but it works well now.

Related question: Should the `POST` handler also ignore those fields? I bet that it should, and it would be no trouble to add.

I ask because Hendrik's [spreadsheet](https://docs.google.com/spreadsheets/d/1OXHJcego9CINwWSJfEAo8aTJ7jZDbhvXshYsEcMuato/edit?usp=sharing) has transcription URLs for a lot of books not on the wanted list (and a few notes, too). If we're going to ignore those fields, I'll ask him to not add them for books not on the wanted list. Or maybe it's fine to just leave them and we can always edit the placeholders later.

Reply:
Yes, if `ebook-placeholder-is-wanted` is unchecked then we should not fill those values from POST.

Reply:
> Yes, if `ebook-placeholder-is-wanted` is unchecked then we should not fill those values from POST.

Got it, done.